app/models/share.rb

```diff
@@ -10,4 +11,14 @@ class Share < ActiveRecord::Base
   has_many :votes, dependent: :destroy
   has_many :comments, dependent: :destroy
 
+  private
+
+  def push_to_firebase
+    response = FIREBASE.push("shares", {
```

Comment:
I think we don't want to do this if it's running our tests.   I think this should be:

```
def push_to_firebase
  if ! Rails.env.test?
response = FIREBASE.push("shares", {
      :title => self.title,
       :body => self.body,
       :lat => self.lat,
       :lon => self.lon 
     })
  end
end
```